spot_ros2_control/xacro/spot_interfaces.ros2_control.xacro

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+    <!-- The interfaces in this are modeled after the parameters from low level control in the spot sdk. -->
+    <xacro:macro name="ros2_control_joint"
+        params="joint_name position_min:=0.0 position_max:=0.0 velocity_max effort_max">
+        <joint name="${joint_name}">
+            <state_interface name="position" />
+            <state_interface name="velocity" />
+            <state_interface name="effort" />
+            <state_interface name="position_commanded" />
+            <state_interface name="velocity_commanded" />
+            <state_interface name="effort_commanded" />
+            <state_interface name="k_q_p_commanded" />
+            <state_interface name="k_qd_p_commanded" />
+
+            <command_interface name="position">
+                <param name="min">${position_min}</param>
+                <param name="max">${position_max}</param>
+            </command_interface>
+            <command_interface name="velocity">
+                <param name="min">${-velocity_max}</param>
+                <param name="max">${velocity_max}</param>
+            </command_interface>
+            <command_interface name="effort">
+                <param name="min">${-effort_max}</param>
+                <param name="max">${effort_max}</param>
+            </command_interface>
+            <command_interface name="k_q_p" />
+            <command_interface name="k_qd_p" />
+        </joint>
+    </xacro:macro>
+
+    <!-- TODO We do not know the real velocity bounds for Spot. -->
```

Comment:
Still going back and forth with BD about this...

Reply:
:exploding_head: